css/style.css

```diff
@@ -46,12 +46,9 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     
+     font-family:'Open Sans', sans-serif;
```

Comment:
watch your indentation. typically just one tab to indent properties inside of your selectors.